.github/actions/setup/action.yaml

```diff
@@ -17,13 +21,22 @@ runs:
       # renovate: datasource=github-tags depName=k3d-io/k3d versioning=semver
       run: curl -s https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | TAG=v5.6.3 bash
 
+      # Homebrew is the preferred installation method for UDS
+      # but homebrew currently doesn't support ARM64 on linux
+      # https://docs.brew.sh/Homebrew-on-Linux#arm-unsupported
     - name: Set up Homebrew
+      if: ${{ runner.arch != 'ARM64' }}
       uses: Homebrew/actions/setup-homebrew@master
 
     - name: Install UDS CLI
+      if: ${{ runner.arch != 'ARM64' }}
       shell: bash
-      # renovate: datasource=github-tags depName=defenseunicorns/uds-cli versioning=semver
-      run: brew install defenseunicorns/tap/uds@0.11.0
+      run: brew install defenseunicorns/tap/uds@${{ inputs.udsCliVersion }}
+
+    - name: Install UDS CLI
+      if: ${{ runner.arch == 'ARM64' }}
+      shell: bash
+      run: curl -o /usr/local/bin/uds -L https://github.com/defenseunicorns/uds-cli/releases/download/v${{ inputs.udsCliVersion }}/uds-cli_v${{ inputs.udsCliVersion }}_Linux_arm64 && chmod +x /usr/local/bin/uds
```

Comment:
would we be able to install the arm64 version by taking `runner.arch` and making it lowercase - GH doesn't have a native option for that (I don't think) but you could pass the arch into the `env` and use bash parameter expansion:

https://stackoverflow.com/questions/62627931/github-actions-expression-functions-string-manipulation

Reply:
I tried this but the amd64 runner.arch is actually "X64" and doesn't resolve. Just thinking we could do a ternary on it though and keep this to one command it skip installing brew.

Reply:
Misread your comment but trying a couple things.

Reply:
Cool yeah I think that works and is much cleaner than the homebrew stuff.